apps/prairielearn/src/batched-migrations/20250214035153_course_instance_usages__submissions__backfill.sql

```diff
@@ -0,0 +1,26 @@
+-- BLOCK delete_old_usages
+DELETE FROM course_instance_usages
+WHERE
+  type = 'Submission'
+  AND date < $CUTOFF_DATE;
+
+-- BLOCK select_bounds
```

Comment:
I checked and this query is fast in prod.

Reply:
Thank you `date` index!